core/src/apps/evolu/evolu_sign_registration_request.py

```diff
@@ -0,0 +1,88 @@
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from trezor.messages import EvoluRegistrationRequest, EvoluSignRegistrationRequest
+
+
+async def evolu_sign_registration_request(
+    msg: EvoluSignRegistrationRequest,
+) -> EvoluRegistrationRequest:
+    from trezor import utils, wire
+    from trezor.crypto.der import read_length
+    from trezor.crypto.hashlib import sha256
+    from trezor.messages import EvoluRegistrationRequest
+    from trezor.utils import BufferReader, bootloader_locked
+
+    from apps.common.writers import write_compact_size
+
+    from .common import check_delegated_identity_proof
+
+    if not bootloader_locked():
+        raise wire.ProcessError(
+            "Cannot sign registration request since bootloader is unlocked."
+        )
+
+    if utils.USE_OPTIGA:
+        from trezor.crypto import optiga
+    else:
+        raise RuntimeError("Optiga is not available")
+
+    if not check_delegated_identity_proof(
+        proposed_value=bytes(msg.proof_of_delegated_identity),
+        header=b"EvoluSignRegistrationRequest",
+        arguments=[
+            bytes(msg.challenge_from_server),
```

Comment:
```suggestion
        proposed_value=msg.proof_of_delegated_identity,
        header=b"EvoluSignRegistrationRequest",
        arguments=[
            msg.challenge_from_server,
```

remove `bytes` conversion, change the called function's parameter types instead

Reply:
this should be solved in here [`ad1baf`](https://github.com/trezor/trezor-firmware/pull/5770/commits/ad1baf00bcaa5c483463cd3e7b52b487a0ced0ab)